ner_v2/detectors/numeral/number/zh-TW/number_detection.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import absolute_import
+import os
+
+try:
+    import regex as re
+
+    _re_flags = re.UNICODE | re.V1 | re.WORD
+except ImportError:
+    import re
+
+    _re_flags = re.UNICODE
+
+from ner_v2.constant import LANGUAGE_DATA_DIRECTORY
+from ner_v2.detectors.numeral.constant import NUMBER_DETECTION_RETURN_DICT_SPAN, \
+    NUMBER_DETECTION_RETURN_DICT_UNIT, NUMBER_DETECTION_RETURN_DICT_VALUE
+from ner_v2.detectors.numeral.number.standard_number_detector import BaseNumberDetector
+
+
+class NumberDetector(BaseNumberDetector):
+    """
+    Number detector to detect numbers in chinese text
+    It map the chinese to latin character as per data file and extract the numeric values
+    """
+
+    data_directory_path = os.path.join((os.path.dirname(os.path.abspath(__file__)).rstrip(os.sep)),
+                                       LANGUAGE_DATA_DIRECTORY)
+
+    def __init__(self, entity_name='number', unit_type=None):
+        super(NumberDetector, self).__init__(entity_name=entity_name,
+                                             data_directory_path=NumberDetector.data_directory_path,
+                                             unit_type=unit_type)
+
+        self.base_numbers_map_full = self.base_numbers_map.copy()
+        self.base_numbers_map_choices_full = self._get_base_map_choices(self.base_numbers_map_full)
+
+        self._filter_base_numbers_map()
+        self.base_numbers_map_choices = self._get_base_map_choices(self.base_numbers_map)
+
+        self.detector_preferences = [
+            self._detect_number_from_text
+        ]
+
+        self.special_chars_mapping = {
+            ',': '、', # comma character
+            '.': '點' #dian ( period )
```

Comment:
E261 at least two spaces before inline comment
E262 inline comment should start with '# '